plugins/analytics/public/malshare.py

```diff
@@ -59,6 +59,10 @@ def analyze(observable, results):
         links = set()
         json_result = MalshareAPI.fetch(
             observable, results.settings['malshare_api_key'])
+
+        if json_result is None:
+            return list(links)
```

Comment:
Maybe this is clearer since links is an empty set anyways:

```suggestion
            return []
```